fl4health/clients/apfl_client.py

```diff
@@ -101,10 +101,9 @@ def compute_loss(
         return losses
 
     def set_optimizer(self, config: Config) -> None:
-        optimizer_dict = self.get_optimizer(config)
-        assert isinstance(optimizer_dict, dict)
-        self.optimizer = optimizer_dict["global"]
-        self.local_optimizer = optimizer_dict["local"]
+        optimizers = self.get_optimizer(config)
+        assert isinstance(optimizers, dict)
```

Comment:
We should probably assert that the dictionary is of length two and contains the proper keys here?